src/scss/_variables.scss

```diff
@@ -1,69 +1,93 @@
 :root {
-  --color-white-rgb: 255, 255, 255;
-  --color-text: rgb(70, 59, 89);
-  --color-text-rgb: 70, 59, 89;
+  //* text colors *//
+  --color-text: #000000;
+  --color-text-on-primary: hsl(250, 55%, 85%);
+  --color-text-on-secondary: #574944;
+  --color-text-on-secondary-light: #6a5e5a;
+  --color-text-on-secondary-dark: #1e0901;
 
-  --color-text-darkbg: #dcc6e0;
-  --color-text-button: #ae5445;
+  //* primary brand color *//
+  --color-primary: hsl(261, 91%, 42%);
+  --color-primary-light: hsl(250, 37%, 51%);
+  --color-primary-lighter: hsl(250, 38%, 55%);
 
-  --color-heading-darkbg: #fbe4ff;
+  --color-primary-dark: hsl(250, 42%, 38%);
+  --color-primary-darker: hsl(251, 45%, 32%);
```

Comment:
Possible baking issue if you need even darker. Maybe dark-1, dark-2?

Reply:
Possibly! I'll keep it as it is for now